client/src/Dashboard/DashboardTitle.js

```diff
@@ -49,6 +51,53 @@ export default function DashboardTitle({ dates, userState, getDashboardData }) {
     </Select>
   )
 
+  const showPaymentModal = () => {
+    setPaymentModalVisible(true)
+  }
+
+  const handlePaymentModalCancel = () => {
+    setPaymentModalVisible(false)
+  }
+
+  const dashboardActions = (
+    <Menu>
+      <Menu.Item key="addAttendance">
+        <Button onClick={() => history.push('/attendance')} type="text">
+          {t('addAttendance')}
+        </Button>
+      </Menu.Item>
+      <Menu.Item key="recordPaymentButtonq">
+        <Button type="text" onClick={showPaymentModal}>
+          {t('recordPaymentButton')}
+        </Button>
+      </Menu.Item>
+    </Menu>
+  )
+
+  const dashboardActionDropdown = (
+    <Dropdown
+      overlay={dashboardActions}
+      className="ml-auto flex"
+      trigger="click"
+    >
+      <Button type="primary">
+        {t('recordDropdown')} <LeftOutlined />
```

Comment:
The figma shows that the Icon should be an arrow pointed down when the dropdown is open. Can you add logic to account for this?

Reply:
I may be looking at the wrong figma, but I do not see this. Is this something we want to do anyways?
![image](https://user-images.githubusercontent.com/44675440/135312856-61ebb74f-99ff-4528-8997-207ab12b880c.png)

Reply:
Hey @davidcandeau !  Check out this link here: https://www.figma.com/file/uCveVQQnYcw3MsenD1PNcG/P4P-2.0?node-id=5969%3A43540 - this is linked in the description of #1613 

Here's a screenshot of the downward arrow
<img width="189" alt="Screen Shot 2021-09-30 at 9 32 48 AM" src="https://user-images.githubusercontent.com/8364647/135475769-36f7f3d8-e1f5-4426-adfa-a1de8ecee52b.png">

Reply:
Perfect, thanks!